src/components/MenuItem.tsx

```
@@ -0,0 +1,52 @@
+};
+
+function formatPrice(price: number): string {
+  return new Intl.NumberFormat("en-UK", {
+    style: "currency",
+    currency: "GBP",
+  }).format(price);
+}
+
+export function MenuItem({ item }: MenuItemProps) {
+  return item.displayItems.map((displayItem: DisplayItem) => (
+    <Card>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)